<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_in">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>INPUT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_dg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>DG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_ca3cue">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>CA3cue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_ca3cont">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>CA3cont</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_ca1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>CA1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_out">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>OUTPUT</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_back">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_next">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_trace">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trace:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cue size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>47</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Content size: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation time (ms):</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_cuesize">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_contsize">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_simtime">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
